cmd/manager/main.go

```diff
@@ -18,17 +18,33 @@ package main
 
 import (
 	"log"
+	"os"
 
 	"github.com/knative/eventing-sources/pkg/apis"
 	_ "github.com/knative/eventing-sources/pkg/apis/serving/v1alpha1"
 	"github.com/knative/eventing-sources/pkg/controller"
+	"github.com/knative/pkg/logging/logkey"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 )
 
+const (
+	ConfigMapNameEnv = "CONFIG_LOGGING_NAME"
```

Comment:
Is this env var set anywhere?

Reply:
Fixed in commit b18e66e and dropped the various CONFIG_LOGGING_NAME references, just set the string directly based on the source.